### PR TITLE
gnome-extra/gnome-commander: version bump to 1.12.1

### PR DESCRIPTION
**Pull request by turboscholz**
Signed-off-by: Uwe Scholz <u.scholz83@gmx.de>

### CODE REVIEW

**Comment by turboscholz** [comment]
The check above complains about the use of deprecated gnome-vfs. I know. Gnome Commander is still depending on it, and I as the maintainer of that program are actively working on getting rid of it. But I cannot say when this work will be finished.

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:4]: Please use the opportunity to bump to `EAPI-7`.

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:37]: Porting to `EAPI-7` would entail moving that block into `BDEPEND` except for `dev-cpp/gtest` (where I expect header use?).

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:62]: You could inherit `optfeature.eclass` here instead. Example line:
`optfeature "synchronizing files and directories" dev-util/meld`

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:22]: this is one line breaking our "sort alphanumerically, unconditional before conditional deps" guideline

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:28]: and this is the second one ;)

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:1]: Need to fix Copyright. You should set yourself up with `repoman` or `pkgdev`/`pkgcheck` to have these things fixed automatically for you plus get initial QA checks.

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:49]: I prefer `local myeconfargs=( ... )` array style these days because it is less error prone, but I'm not forcing your hand on that matter.

**Review by a17r — CHANGES_REQUESTED** [review]

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (turboscholz) [reply gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:1]: Thank you for the thorough review. To be honest I never really used these tools up to now. But when trying a `repoman -dx full` it just complains about the deprecated dependency:
```
$ repoman -dx full

RepoMan scours the neighborhood...
  dependency.deprecated         6
   gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild: '>=gnome-base/gnome-vfs-2.0.0'
   gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild: '>=gnome-base/gnome-vfs-2.0.0[samba]'
```
Is this expected? The other issues which you mentioned like the date in line 1, the ordering and all that were not mentioned by repoman.

Comment (a17r) [reply gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild]: Yes, of those tools `pkgcheck` would be the most thorough; `repoman fix` would fix basic QA issues and `repoman commit` prepares half of a git workflow compliant git summary for you to complete. Both don't mind style best practises that we communicate as part of our review.

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:41]: Ah, not quite. `BDEPEND` is the one *not* inheriting from either `DEPEND` and `RDEPEND`.
`DEPEND` needs to inherit `RDEPEND` just as it did before.

Please stick to one style - I'd say if you put quotes on a separate line, as I prefer too, then an empty line before `DEPEND` is unnecessary, just as it wasn't there before.

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:65]: Did you check the output on a test run? I didn't include the example line without reason. ;)

**Review by a17r — CHANGES_REQUESTED** [review]

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:59]: this should be first in line of the phase functions:

https://devmanual.gentoo.org/ebuild-writing/functions/index.html

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:8]: please sort them (of those, only gnome2 is exporting phase functions, so you are safe).

File: gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild
Comment (a17r) [inline gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild:16]: please sort them too ;)

**Review by a17r — CHANGES_REQUESTED** [review]
Last nitpicks before merge!

**Comment by turboscholz** [comment]
Great, thanks again for the thorough review! :+1:

**Pull request closed without merging**
